templates/web/base/waste/bin_days.html

```diff
@@ -108,19 +108,21 @@ <h3>Download your collection schedule</h3>
        [% SET show_garden_subscribe = NOT waste_features.garden_disabled AND NOT pending_subscription AND NOT services_available.Garden_Waste.is_active %]
        [% IF any_report_allowed OR any_request_allowed OR show_garden_subscribe %]
         <div class="aside-services">
-          <h3>More services</h3>
-          <ul>
-            [% IF any_report_allowed %]
-              <li><a href="[% c.uri_for_action('waste/report', [ property.id ]) %]">Report a missed collection</a></li>
-            [% END %]
-            [% IF any_request_allowed %]
-              <li><a href="[% c.uri_for_action('waste/request', [ property.id ]) %]">Request a new container</a></li>
-            [% END %]
-            [% IF show_garden_subscribe %]
-              <li><a href="[% c.uri_for_action('waste/garden', [ property.id ]) %]">Subscribe to Green Garden Waste collection</a></li>
-            [% END %]
-            [% TRY %][% PROCESS waste/_more_services_extra.html %][% CATCH file %][% END %]
-          </ul>
+          [% TRY %][% PROCESS waste/_more_services_sidebar.html %][% CATCH file %]
+            <h3>More services</h3>
+            <ul>
+              [% IF any_report_allowed %]
+                <li><a href="[% c.uri_for_action('waste/report', [ property.id ]) %]">Report a missed collection</a></li>
+              [% END %]
+              [% IF any_request_allowed %]
+                <li><a href="[% c.uri_for_action('waste/request', [ property.id ]) %]">Request a new container</a></li>
+              [% END %]
+              [% IF show_garden_subscribe %]
+                <li><a href="[% c.uri_for_action('waste/garden', [ property.id ]) %]">Subscribe to Green Garden Waste collection</a></li>
+              [% END %]
+              [% TRY %][% PROCESS waste/_more_services_extra.html %][% CATCH file %][% END %]
```

Comment:
This was only used by Peterborough so I guess you could drop it here and in Peterborough sidebar.html and include its contents directly in sidebar.html?